packages/grafana-ui/src/components/Forms/InlineField.mdx

```diff
@@ -0,0 +1,16 @@
+import { Props } from "@storybook/addon-docs/blocks";
+import { InlineField } from "./InlineField";
+
+# InlineField
+
+A basic component for rendering inline form elements together with their label. If the child element has `id` specified, the label's `htmlFor` attribute, pointing to the id, will be added.
```

Comment:
Nice starting point. I think it's useful to list which inline form elements you could use to render with the label, and also give related component names here for 100% clarity (e.g. not just say "together with their label" but "together with `InlineLabel`", mention `InlineFieldRow`).

> If the child element has `id` specified, the label's `htmlFor` attribute, pointing to the id, will be added.
The phrasing is a bit complicated; maybe say "…specified, the component automatically gets a `htmlFor` attribute which points to the id". Also I don't yet understand why this information is important or why the label needs an attribute pointing to the id.

Maybe it's a good idea to explain how you can tweak the component to fit your needs, e.g. that you can modify the width of labels and/or fields using [list of all props that do this], what it looks like when the children prop is used, etc.

Can I have one label with multiple fields? If yes, how do I do that?